moveit_plugins/moveit_simple_controller_manager/src/moveit_simple_controller_manager.cpp

```diff
@@ -52,7 +52,7 @@ class MoveItSimpleControllerManager : public moveit_controller_manager::MoveItCo
   {
     if (!node_handle_.hasParam("controller_list"))
     {
-      ROS_ERROR_STREAM_NAMED("manager", "No controller_list specified.");
+      ROS_ERROR_STREAM_NAMED("moveit_simple_controller_manager", "No controller_list specified.");
```

Comment:
Here we don't need an additional name at all. `moveit_simple_controller_manager` is already the package name.